src/Data/NodeResolver.php

```diff
@@ -336,6 +336,20 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 			$this->wp->query_vars['error'] = $error;
 		}
 
+		/**
+		 * Filters the array of parsed query variables.
+		 *
+		 * @param mixed|null $node The node, defaults to nothing.
+		 * @param string $uri The uri being searched.
+		 * @param AppContext $content The app context.
+		 * @param WP $wp WP object.
+		 * @param mixed|array|string $extra_query_vars Any extra query vars to consider
+		 */
+		$node = apply_filters( 'graphql_resolve_uri_query_vars', null, $uri, $this->context, $this->wp, $extra_query_vars );
```

Comment:
@markkelnar When using this pattern of having a `null` and then checking if it's not null and returning, we use `graphql_pre_` as the prefix, stemming from patterns in WordPress core. 

I think this looks good, but I think we should change the name to something like: `graphql_pre_resolve_uri_with_query_vars`